platforms/generic-pi/boot-config-loader/pkg.yaml

```diff
@@ -0,0 +1,17 @@
+name: rpi-boot-config-loader
+variant: scratch
+shell: /toolchain/bin/bash
+dependencies:
+  - stage: base
+  - image: "ghcr.io/ogkevin/talos-ext-generic-pi:{{.VERSION}}"
```

Comment:
this seems like a private repo?

Reply:
It's public 🤔 https://github.com/OGKevin/talos-ext-rpi-generic/pkgs/container/talos-ext-generic-pi

Unless I'm misunderstanding?

Reply:
@frezbo little reminder poke 🙈